### PR TITLE
Fixes #126: Added 2(knowledge) and 1(entertainment) susi skills.

### LINKED ISSUE #126: Susi is giving a non-human answer to the question 'What is your name'
**Actual Behaviour**

Once user types what is your name? 
Susi answers: your name is a 2017 Japanese movie by Shinkai Makoto. 
This answer is correct but user wants to know if susi can tell her name 

**Expected Behaviour**
- My name is Susi.  

**Steps to reproduce it**
- find out where susi get the answer for that question and fix it

### PR DESCRIPTION
**Pull request by Manveer695**

### CODE REVIEW

**Comment by Orbiter** [comment]
please do not make a new topic "enhacementofsusi".
The README states that you should not make new topics because otherwise we will quickly grow in topics and things will be messed up.

There, in enhacementofsusi, you also placed your skill into "es" but your skill is clearly not spanish.
Please put that skill into knowledge, maybe into an existing expert or a new one

**Comment by Manveer695** [comment]
@Orbiter Yes sir. I got to know that in the morning and will do the necessary changes soon. I also wrote that in my today's SCRUM. Also, I haven't put anything in es folder, it's in en folder only. But still enhancementofsusi folder is absolutely not required. I will change it immediately. Thank you.

File: models/general/knowledge/en/who_is.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# answers who is question efficiently
```
Comment (chashmeetsingh) [inline models/general/knowledge/en/who_is.txt]: Hey, can you take a look at https://products.wolframalpha.com/api/explorer/?
This gives a much better response for who+is or what+is? Try that maybe?

**Comment by Orbiter** [comment]
the "who_is.txt" expert creates a skill which is calling itself which should cause an endless loop using a "what is" expert that is already implemented. Thats not good. Please remove the skill with the call to susi api

**Comment by Manveer695** [comment]
@Orbiter Done. Also, I want to raise an issue, that any question in ``(backquotes) doesn't redirect this question to Susi automatically.

**Pull request merged**